pkg/apis/planetscale/v2/vitesscluster_types.go

```diff
@@ -107,6 +107,37 @@ type VitessClusterSpec struct {
 	// not have any prefix (just "flag", not "-flag"). To set a boolean flag,
 	// set the string value to either "true" or "false".
 	ExtraVitessFlags map[string]string `json:"extraVitessFlags,omitempty"`
+
+	// TopoReconciliation can be used to enable or disable registration or pruning of various vitess components to and from topo records.
+	TopoReconciliation TopoReconcileConfig `json:"topoReconciliation,omitempty"`
+}
+
+// TopoReconcileConfig can be used to turn on or off registration or pruning of specific vitess components from topo records.
+// This should only be necessary if you need to override defaults, and shouldn't be required for the vast majority of use cases.
+type TopoReconcileConfig struct {
+	// RegisterCellsAliases can be used to enable or disable registering cells aliases into topo records.
+	RegisterCellsAliases *bool `json:"registerCellsAliases,omitempty"`
```

Comment:
```suggestion
	RegisterCellAliases *bool `json:"registerCellsAliases,omitempty"`
```

Reply:
i'd put this under RegisterCells, since Aliases are presumably less important but related to Cells.

Reply:
The official feature in Vitess is called CellsAlias, and we are talking about the plural (more than one CellsAlias).

Reply:
Cells and CellsAlias are actually very different concepts. The name, unfortunately, makes it sound more similar than it is.